src/pilot/dracclient_raid.patch

```diff
@@ -1,205 +1,31 @@
---- resources/raid.py	2018-09-19 04:11:40.000000000 -0500
-+++ resources/raid_backup.py	2019-01-08 12:15:39.961836000 -0600
-@@ -571,9 +571,201 @@
-                 # Fix for python 3, Exception.message no longer
-                 # a valid attribute, str(ex) works for both 2.7
-                 # and 3.x
--                if self.NOT_SUPPORTED_MSG in str(ex):
-+                if constants.NOT_SUPPORTED_MSG in str(ex):
-                     pass
-                 else:
-                     raise
+--- raid.py.orig	2019-01-23 15:57:50.655000000 +0000
++++ raid_backup.py	2019-01-23 14:44:59.068000000 +0000
+@@ -588,14 +588,26 @@
+         """
+         return raid_controller_fqdd.startswith('RAID.')
  
-         return is_jbod_capable
-+
-+    def is_raid_controller(self, raid_controller_fqdd):
```

Comment:
Nice job eliminating these methods from the patch since they are now available in OSP13z4!